containers/images/pulp-celerybeat/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM quay.io/foreman/pulp:latest
+
+COPY container-assets/entrypoint.sh /usr/bin/entrypoint.sh
+COPY container-assets/start_pulp_celerybeat.sh /usr/bin/start_pulp_celerybeat.sh
+
+USER apache
+
+CMD ["/usr/bin/start_pulp_celerybeat.sh"]
+ENTRYPOINT ["/usr/bin/entrypoint.sh"]
```

Comment:
Are any of these process based where they would need a better init here?

Reply:
Can you expand for me on 'better init' ?

Reply:
[dumb-init](https://github.com/Yelp/dumb-init) or [tini](https://github.com/krallin/tini) specifically something that handles signals and zombie reaping properly.

Reply:
I'm just not sure which of these containers would have child processes which exit and would need to  be `wait()`ed on.

Reply:
Right, if this is just a translation then this is fine for now.

I'll comment on the issue around our experience with dumb-init.